docs/references.bib

```diff
@@ -138,6 +138,13 @@ @book{wall2018analytic
   publisher={Dover Publications}
 }
 
+@book{hardy1979introduction,
```

Comment:
Any particular reason to cite 5th edition? It seems that the latest edition is 6th, see https://global.oup.com/academic/product/an-introduction-to-the-theory-of-numbers-9780199219865?cc=us&lang=en&

Reply:
It's just the edition I had at my hands (: I updated the reference as it still contains the relevant chapter.